server/api/users/users.controller.ts

```diff
@@ -18,7 +18,15 @@ export const getCurrentUser: RequestHandler<void, ClientUser>  = (req, res) => {
   res.json(filteredUser);
 };
 
-export const getUserById: RequestHandler<{id: string}, UserProfile> = async (req, res) => {
+interface UpdatableFields {
+  bio?: string;
+  twitter_username?: string;
+  linkedin_url?: string;
+  github_username?: string;
+  website?: string;
+}
```

Comment:
nit: let's colocate this with the controller method iself; i.e. put above `updateUserById`